### ISSUE TITLE
Room Directory Integration

### ISSUE THREAD

**Issue by travankor** [mirukana/mirage#57]
UI to search rooms on homeservers like Riot.

Similarily, have a UI when creating rooms for publishing in room directory and setting addresses.

**Comment by mirukana** [comment]
Addresses and publish room settings should be easy to add, will land soon in one of the 0.5.x release.

Room search UI is more complicated, needing work in all areas (nio support, mirage backend and controls), so it's not in my priorities, especially with the current state of the directory (https://github.com/vector-im/riot-web/issues/4683).

**Comment by travankor** [comment]
Fair enough, especially if Matrix ends up rewriting how room search works. The main reason I found this useful is that I could not directly connect to room ID's (ie: !string) and had to use the web interface to get to the room. The room had no addresses and might be an old room version, which might be why it didn't work.
